wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -293,7 +293,7 @@ <h3>Buttons with internal loading indicators</h3>
 
             <h4>Buttons where the text is replaced on click</h4>
             <button class="button button-longrunning" data-clicked-text="Test" type="button">{% icon name="spinner" %}<em>Click me</em></button>
```

Comment:
Could you update the above buttons to use the controller, and use a custom `data-w-loader-duration-seconds-value="5"` so it’s easier for us to test with them.